utils/mock-data-loader/.env.sample

```diff
@@ -1,3 +1,6 @@
 DEAL_API_URL=http://localhost:5001
 DTFS_CENTRAL_API=http://localhost:5005
-TFM_API=http://localhost:5004
\ No newline at end of file
+TFM_API=http://localhost:5004
+
+MONGO_INITDB_DATABASE=
```

Comment:
Once this is merged devs will need to fill in these values. They should already have the values from other `.env` files. I've made a note to post on Slack when this is merged